src/pages/terms-and-conditions/_business-grid.tsx

```diff
@@ -62,7 +62,14 @@ const PartnersGuidePdf = () => {
 }
 
 const BusinessGrid = () => {
+    const general_terms_url = `/tnc/business-partners-general-terms-es.pdf`
```

Comment:
Could put **_es** or **_spanish** at the end of these variable names for clarity

Reply:
Other Pdfs which is translated is named like this..so as a common gave the name the same

Reply:
Pdf file is okay, I meant this **general_terms_url** maybe to **general_terms_url_es**

Reply:
In future,other languages will also come up..so we should use a constant variable..should not name it based on the language..only the url link will change..but the variable name should remain constant for all the languages